src/Schema/Tables/Appointments/Appointments.sql

```diff
@@ -0,0 +1,16 @@
+﻿CREATE TABLE [dbo].[Appointments] 
+  ( 
+     [AppointmentId]   INT NOT NULL IDENTITY(1, 1), 
+     [PatientId]       INT NOT NULL, 
+     [UserId]          INT NOT NULL, 
+     [AppointmentDate] DATE NOT NULL DEFAULT Getdate(), 
```

Comment:
@DenyaAbramenkov please check what date types exist in the sql and double check that you have selected right one for appointments.